_posts/2022-12-05-Introducing-Identity.markdown

```diff
@@ -0,0 +1,40 @@
+# Introducing Identity for OpenSearch
+
+OpenSearch handles critical data; how that data is permissioned and how a cluster is operated is a Big Deal™.  Consider the question "Who can delete all the indices in the cluster?", in the core of OpenSearch, the answer is whoever can make a request to the cluster.  

```

Comment:
I'm not sure what "Big Deal™" is. I googled this term and didn't get any hits that made sense.

Reply:
Dropped this kind of tone and language.